src/xdocs-examples/resources/com/puppycrawl/tools/checkstyle/checks/coding/textblockgooglestyleformatting/Example1.java

```diff
@@ -0,0 +1,48 @@
+/*xml
+<module name="Checker">
+  <module name="TreeWalker">
+    <module name="TextBlockGoogleStyleFormatting"/>
+  </module>
+</module>
+*/
+package com.puppycrawl.tools.checkstyle.checks.coding.textblockgooglestyleformatting;
+
+// xdoc section -- start
+public class Example1 {
```

Comment:
please add code with violation.

Follow a patter as in other Examples: bad code chunk, and under it expected good code chunk.

show users all violation messages.